coalib/coala.py

```diff
@@ -34,9 +30,11 @@ def main():
         args = default_arg_parser().parse_args()
 
         if args.show_bears:
+            from coalib.settings.ConfigurationGathering import get_filtered_bears
             local_bears, global_bears = get_filtered_bears(
                 args.filter_by_language, log_printer)
 
+            from coalib.output.ConsoleInteraction import show_bears
```

Comment:
But I guess that doesn't give a big performance gain (we should profile extensively)